docs/config/README.md

```diff
@@ -40,6 +40,15 @@ are segments within it. Every module also has a prefix and suffix that are the d
  "via "         "⬢"        "v10.4.1"       ""
 ```
 
+### Style Strings
+
+Most modules in starship allow you to configure their display styles. This is done with an entry (usually called `style`, but some modules have different names) which is a string. Here are some examples of style strings along with what they do. For details on the full syntax, consult the advanced config guide.
```

Comment:
How do I link this to the actual advanced config guide?

Reply:
It looks like the syntax would be `[advanced config guide](/advanced-config/)`

https://vuepress.vuejs.org/guide/markdown.html#links